### PR TITLE
Feature/minitest installer

### PR DESCRIPTION
**Pull request by TylerRockwell**
### Why?
Because some developers use minitest, and they should have an autoinstaller too.

### What Changed?
Added minitest installer to `install_generator`
Added tests for new installation
Updated documentation
Bumped gem version

### CODE REVIEW

File: gem_rspec/install_generator_spec.rb [hunk 1]
```diff
@@ -27,17 +32,49 @@
...
+
+  context "when minitest is installed" do
+    describe "#configure_smashing_docs" do
+      let(:test_helper) { "test/test_helper.rb" }
+      it "appends the SmashingDocs config to test_helper.rb" do
+        `mv spec s`
+        run_generator
+        expect(File).to exist(test_helper)
+        expect(File.read(test_helper)).to include("SmashingDocs.config do")
+        `rm #{test_helper}`
```
Comment (jahammo2) [inline gem_rspec/install_generator_spec.rb]: Are you actually running a command line command here?

File: lib/generators/smashing_documentation/install_generator.rb [hunk 2]
```diff
@@ -6,49 +6,91 @@ class InstallGenerator < Rails::Generators::Base
...
+          config = "SmashingDocs.config do |c|\n"\
+                   "  c.template_file = 'smashing_docs/template.md'\n"\
+                   "  c.output_file   = 'smashing_docs/api_docs.md'\n"\
+                   "  c.auto_push     = false\n"\
+                   "end\n"
+          if using_minitest?
+            insert_into_file(@config_file, config, after: "class ActiveSupport::TestCase\n")
+          else
+            append_file(@config_file, config)
+          end
```
Comment (jahammo2) [inline lib/generators/smashing_documentation/install_generator.rb]: Any time I find myself doing nested conditionals, I always ask myself if there is a way to refactor. It looks like there is an easy way.

```
      def add_configuration
        unless config_already_setup?
          setup_config
        end
      end

      ...

      def setup_config
        config = "SmashingDocs.config do |c|\n"\
                 "  c.template_file = 'smashing_docs/template.md'\n"\
                 "  c.output_file   = 'smashing_docs/api_docs.md'\n"\
                 "  c.auto_push     = false\n"\
                 "end\n"
        if using_minitest?
          insert_into_file(@config_file, config, after: "class ActiveSupport::TestCase\n")
        else
          append_file(@config_file, config)
        end
      end
```

File: gem_rspec/install_generator_spec.rb (hunk 1 above)
Comment (TylerRockwell) [inline gem_rspec/install_generator_spec.rb]: I am.

**Comment by TylerRockwell** [comment]
Thanks, @jahammo2 !

**Pull request merged**